source/features/reactions-avatars.tsx

```diff
@@ -4,10 +4,9 @@ import select from 'select-dom';
 import debounce from 'debounce-fn';
 import {timerIntervalometer} from 'intervalometer';
 import features from '../libs/features';
-import {getUsername, flatZip} from '../libs/utils';
+import {getUsername} from '../libs/utils';
 
-const arbitraryAvatarLimit = 36;
-const approximateHeaderLength = 3; // Each button header takes about as much as 3 avatars
+const approximateAvatarCount = 16; // Each row takes about as much as 16 avatars.
```

Comment:
This is more self-explanatory

```suggestion
const avatarRowLimit = 16;
```